server/src/main/java/org/elasticsearch/watcher/FileWatcher.java

```diff
@@ -199,10 +227,16 @@ private void init(boolean initial) throws IOException {
             }
         }
 
-        private FileObserver createChild(Path file, boolean initial) throws IOException {
-            FileObserver child = new FileObserver(file);
-            child.init(initial);
-            return child;
+        private Observer createChild(Path file, boolean initial) throws IOException {
+            try {
+                FileObserver child = new FileObserver(file);
+                child.init(initial);
+                return child;
+            } catch (AccessControlException e) {
+                // don't have permissions, use a placeholder
+                logger.debug("Don't have permissions to watch path [{}]", file);
```

Comment:
Does this warrant info logs? And do we want to know what's accessing these? In that case the stacktrace might be helpful

Reply:
I forgot to log the exception...

My concern about the logs is that this happens regularly with FileWatcher, as a watch on a directory tries to watch all files, so this would spam the log file

Reply:
So it looks like this only logs when file or directory is created - but there are several watchers, which do spam the logs on startup. I think leaving it as debug is ok for now